packages/okidoc-md/src/buildDocumentationSource/buildDocumentationSourceAST/buildDocumentationSourceAST.js

```diff
@@ -24,38 +25,66 @@ function buildDocumentationSourceAST({
     createApiFunction,
   } = getVisitorApi({ tag, visitorPath });
 
-  const interfaces = [];
-  const classDeclarations = [];
-  const classBodyItems = [];
-  const functions = [];
-
   // NOTE: read about visitors
   // https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#toc-visitors
   // https://github.com/babel/babel/blob/master/packages/babel-traverse/src/visitors.js
-  const visitors = {
-    'InterfaceDeclaration|TSInterfaceDeclaration': (path, state) => {
-      const interfaceDeclaration = createApiInterface(path.node, path);
+  const interfaceVisitor = {
+    'GenericTypeAnnotation|TSTypeReference': (path, state) => {
+      const file = state.dependenciesTree.get(state.filePath);
+
+      if (file) {
+        // flow support with GenericTypeAnnotation, how to verify?
+        const typeName = t.isTSTypeReference(path.node) ? path.node.typeName.name : path.node.id.name;
+        // NOTE: it might be the case when more than 1 import exists.
+        // Example:
+        // import { MyType } from './types';
+        // import { MyType } from './another-type';
```

Comment:
Is not it invalid code if there two imports with the same specifier name?

Reply:
Hm, I was thinking such sort of validation have to be done eslint / tslint toos. 
But it's not true - typescript is throwing an error in such cases `TS2300: Duplicate identifier 'Type'.`

But in the other hand next code should be valid:
```
// @ts-ignore
import { Type } from './type1.ts';
// @ts-ignore
import { Type } from './type2.ts';
```